src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color, transform 300ms linear;
```

Comment:
Checklist p.4 use variables for the main values so that you'll be able to reuse them and give them descriptive names....
I think it's better to create a variable for the '300ms' value, fix in all cases (c)Ihor Karpyn)))